core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -675,6 +685,42 @@ public static boolean defaultUseHttp1Pipelining() {
         return DEFAULT_USE_HTTP1_PIPELINING;
     }
 
+    /**
+     * Returns the default value. If true then HTTP 2.0 PING frames are sent on connection, no otherwise.
+     * Note that if this is set to true, you may use {@link Flags#defaultHttp2PingTimeoutMillis()} to tune
+     * the timeout interval. The default is 3 seconds.
+     *
+     * <p>This flag is disabled by default. Specify the
+     * {@code -Dcom.linecorp.armeria.defaultUseHttp2PingOnIdleConnection=true} JVM option to enable it.
+     */
+    public static boolean defaultUseHttp2PingOnIdleConnection() {
+        return DEFAULT_USE_HTTP2_PING_ON_IDLE_CONNECTION;
+    }
+
+    /**
+     * Returns the default value. If true then HTTP 2.0 PING frames are sent even there are no active
+     * HTTP 2.0 stream, no otherwise.
+     *
+     * <p>This flag is disabled by default. Specify the
+     * {@code -Dcom.linecorp.armeria.defaultUseHttp2PingOnIdleStream=true} JVM option to enable it.
+     */
+    public static boolean defaultUseHttp2PingOnIdleStream() {
```

Comment:
Oh, I misunderstood how this flag works when I suggested a rename. `defaultUseHttp2PingOnNoActiveStreams` is correct indeed. Sorry about giving a wrong suggestion. :bowing_man:

Reply:
No problem 😄